salt/preflight/defaults.yaml

```diff
@@ -0,0 +1,32 @@
+destructive: False      # define if /var/lib/etcd if should be removed
```

Comment:
Still not convinced this 'state' should ever do something 'destructive' :smiley:

Reply:
I do agree.
Having a state that destroy things can exists with safety nets, and possibly no way to be called besides explicitly calling it. Never call it in a highstate.